clients/cli/src/test/java/org/apache/gravitino/cli/TestTableCommands.java

```diff
@@ -41,35 +46,52 @@
 import org.apache.gravitino.cli.commands.TableSortOrder;
 import org.apache.gravitino.cli.commands.UpdateTableComment;
 import org.apache.gravitino.cli.commands.UpdateTableName;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 class TestTableCommands {
   private CommandLine mockCommandLine;
   private Options mockOptions;
 
+  private final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+  private final ByteArrayOutputStream errContent = new ByteArrayOutputStream();
+  private final PrintStream originalOut = System.out;
+  private final PrintStream originalErr = System.err;
+  private static final String METALAKE = "metalake_demo";
+  private static final String CATALOG = "catalog";
+  private static final String SCHEMA = "SCHEMA";
+  private static final String TABLE = "users";
+
```

Comment:
There is no need to do this, while typically, code should be DRY (don't repeat yourself), test code can be WET, and we could keep the same style as other tests.